src/formio/components/addressNL.ts

```diff
@@ -24,4 +34,5 @@ export interface AddressNLComponentSchema
   type: 'addressNL';
   deriveAddress: boolean;
   layout: 'singleColumn' | 'doubleColumn';
+  components?: AddressComponents;
```

Comment:
I'm not sure if I can make this required, if I do that, does that mean we have to run data migrations to add this property to existing addressNL components?